src/run_integration_tests.h

```diff
@@ -20,7 +20,7 @@
  */
 /*@{*/
 /**
- * @file run_integration_test.h
+ * \@file run_integration_test.h
```

Comment:
Is the backslash necessary here? Or is it a symptom of above /*@{*/?

It's my understanding that Doxygen treats both `\` and `@` as "start Doxygen directive", I don't understand why we'd need both.

Reply:
The doxygen returns a waning:
`warning: the name 'run_integration_test.h' supplied as the argument in the \file statement is not an input file`
 
Current solution is found from Stackoverflow:
https://stackoverflow.com/questions/32872881/markdown-in-doxygen-with-file

Reply:
'run_integration_test.h' -> 'run_integration_test**s**.h'

Reply:
Typo: `run_integration_test.h` -> `run_integration_tests.h`